payments/models.py

```diff
@@ -105,7 +124,12 @@ class Rate(models.Model):
     price = models.DecimalField(max_digits=6, decimal_places=2,
```

Comment:
The `max_digits` needs to be bumped up to 8, I think:

![image](https://cloud.githubusercontent.com/assets/274668/18528470/07a24f76-7a7e-11e6-85e4-9d947b5a841d.png)